src/commands/update.rs

```diff
@@ -27,6 +28,8 @@ pub(crate) fn update(command: &Command) -> Status {
     if !utils::functions::check_internet_connection() {
         return Status::error("You need a internet connection to update templify.".to_string());
     }
+    let mut current_version = VersionNumber::new();
+    current_version.parse_from_string(env!("CARGO_PKG_VERSION"));
```

Comment:
Both lines can be moved after the if branch below. This saves the computation of the parsing when updating is not possible

Reply:
Hi @cophilot , Thanks for pointing this out, Fixed this issue in my new push [commit](https://github.com/cophilot/templify/pull/12/commits/1dfe74d7a60718119744a66e6eee16684293f344)